app/views/certificate/vatCertificate.scala.html

```diff
@@ -49,36 +66,13 @@ <h1 class="heading-xlarge">@if(user.isAgent){@messages("vatCertificate.agentTitl
             <h2 class="heading-medium card-heading">@messages("vatCertificate.aboutYourRegistration.title")</h2>
         </div>
         <div class="column-full card-full">
-            <dl class="govuk-check-your-answers cya-questions-long">
-                <div>
-                    <dt class="cya-question">
-                        @messages("vatCertificate.aboutYourRegistration.vrn")
-                    </dt>
-                    <dd class="cya-answer">
-                        @model.vrn
-                    </dd>
-                </div>
-            </dl>
-            <dl class="govuk-check-your-answers cya-questions-long">
-                <div>
-                    <dt class="cya-question">
-                        @messages("vatCertificate.aboutYourRegistration.registrationDate")
-                    </dt>
-                    <dd class="cya-answer">
-                        @{model.registrationDate.map(displayDate(_)).getOrElse(messages("common.notProvided"))}
-                    </dd>
-                </div>
-            </dl>
-            <dl class="govuk-check-your-answers cya-questions-long">
-                <div>
-                    <dt class="cya-question">
-                        @messages("vatCertificate.aboutYourRegistration.certificateDate")
-                    </dt>
-                    <dd class="cya-answer">
-                        @displayDate(model.certificateDate)
-                    </dd>
-                </div>
-            </dl>
+            @fieldRow(messages("vatCertificate.aboutYourRegistration.vrn"), model.vrn)
```

Comment:
Can you double check this. The acceptance criteria does state that it should appear in the 'About Your Registration' section but all of the prototypes show it under 'About the business'.